lib/middleware/redirects.js

```diff
@@ -54,15 +61,15 @@ Redirect.prototype.test = function(url) {
 
       return {
         type: this.type,
-        destination: dest
+        destination: dest + qs
```

Comment:
Let's keep it simple -- I'm now appending them, including a dup if there's a dup between the supplied qs and the destination qs.